src/streams/appendToStream/unpackError.ts

```diff
@@ -0,0 +1,81 @@
+import { Status } from "../../../generated/status_pb";
+import {
+  AccessDenied,
+  MaximumAppendSizeExceeded,
+  StreamDeleted,
+  Timeout,
+  Unknown,
+  WrongExpectedVersion,
+} from "../../../generated/shared_pb";
+import {
+  AccessDeniedError,
+  MaxAppendSizeExceededError,
+  StreamDeletedError,
+  TimeoutError,
+  UnknownError,
+  WrongExpectedVersionError,
+} from "../..";
+
+export const unpackWrongExpectedVersion = (grpcError: Status) =>
```

Comment:
**FIX:** I think we're missing tests for the logic in this file, so unpacking the wrong expected version. IMHO, we should add tests for negative scenarios and ensure that we have matching coverage to the regular append.

Reply:
Additional tests for possible error types added here:
https://github.com/EventStore/EventStore-Client-NodeJS/pull/229/commits/22495039f75921504741c5a0825913e55860ece5